src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -69,9 +69,10 @@ ALTER ROLE CURRENT_USER WITH PASSWORD 'dummypassword';
 -- Show that, with no MX tables, activate node snapshot contains only the delete commands,
 -- pg_dist_node entries, pg_dist_object entries and roles.
 SELECT unnest(activate_node_snapshot()) order by 1;
-                                                                                                                                                                                                                                                       unnest
+                                                                                                                                                                                                                         unnest
 ---------------------------------------------------------------------
  ALTER DATABASE regression OWNER TO postgres;
+ CALL pg_catalog.worker_drop_all_shell_tables(1::bool)
```

Comment:
hmm, why don't we have `true` but this weird cast? Can you change `WorkerDropAllShellTablesCommand` to pass true or false?